src/components/HomePage.js

```diff
@@ -0,0 +1,43 @@
+import { useDispatch, useSelector } from 'react-redux';
+import InputSearch from './InputSearch';
+import DisplayCoin from './DisplayCoin';
+import styles from './styles/HomePage.module.css';
+import { coinsActions } from '../redux/coinsSlice';
+
+const HomePage = () => {
+  const coinsArray = useSelector((state) => state.coins.coins);
+  const dispatch = useDispatch();
+
+  const filteredArray = useSelector((state) => state.coins.filteredArray);
+
+  const inputSearchHandler = (e) => {
+    e.preventDefault();
```

Comment:
- [ ] 🙏Please ensure that your design🎉 matches the original design. In the original design, they used a two-column layout for categories, while in your design, you used a single-column layout. By aligning your design with the original, it will give your design a cool🌹 and consistent look.

Your design:
![image](https://github.com/bilalrajput09/CryptoWatch-React-App/assets/63445083/1b6e19fc-647d-4a02-837a-e211cf02f10e)
Original design:
![image](https://github.com/bilalrajput09/CryptoWatch-React-App/assets/63445083/4ec2a40b-6cd9-4951-a606-7066d8dfe46c)

Reply:
Alright ✅.

Reply:
Fixed ✔